packages/adapters/ethers5/src/client.ts

```diff
@@ -257,17 +257,17 @@ export class Ethers5Adapter extends AdapterBlueprint {
     connectors.forEach(connector => {
       const key = connector === 'coinbase' ? 'coinbaseWalletSDK' : connector
 
-      const injectedConnector = connector === CommonConstantsUtil.CONNECTOR_ID.INJECTED
+      const isInjectedConnector = connector === CommonConstantsUtil.CONNECTOR_ID.INJECTED
 
       if (this.namespace) {
         this.addConnector({
           id: key,
           explorerId: PresetsUtil.ConnectorExplorerIds[key],
           imageUrl: options?.connectorImages?.[key],
-          name: PresetsUtil.ConnectorNamesMap[key],
+          name: PresetsUtil.ConnectorNamesMap[key] || key,
```

Comment:
this feels a bit weird, if the `key` is not in `PresetsUtil.ConnectorNamesMap` then are we going to show it as e.g. `coinbaseWalletSDK`? Why don't we have the name here from the extension?